pkg/commands/git_commands/branch_loader.go

```diff
@@ -182,12 +195,18 @@ func obtainBranch(split []string) *models.Branch {
 	track := split[3]
 	subject := split[4]
 	commitHash := split[5]
+	commitDate := split[6]
 
 	name := strings.TrimPrefix(fullName, "heads/")
 	pushables, pullables, gone := parseUpstreamInfo(upstreamName, track)
+	recency := ""
+	if unixTimestamp, err := strconv.ParseInt(commitDate, 10, 64); err == nil {
+		recency = utils.UnixToTimeAgo(unixTimestamp)
+	}
```

Comment:
When sorting by recency, branches that don't have a reflog entry for some reason used to have an empty recency column. Now they show the committer date, which is confusing. We should only set the recency here when not sorting by "recency"; in my approach I solved this by adding a `storeCommitterDateAsRecency` parameter to `obtainBranch`, see https://github.com/jesseduffield/lazygit/commit/268c56bd3e209141b0563747a2b26e419892218e#diff-6daca80788b73b1a2870465493179bde13e0be1ef2612f377f8dddb38dd6986eR193. (Would need another change to the branch_loader tests.)

Reply:
Totally missed that, fixed in https://github.com/jesseduffield/lazygit/pull/3182/commits/bea96a5babac6ae2a941e99a01446d7c23fc2c8e